airbyte-integrations/connectors/destination-redshift/Dockerfile

```diff
@@ -16,5 +16,5 @@ ENV APPLICATION destination-redshift
 
 COPY --from=build /airbyte /airbyte
 
-LABEL io.airbyte.version=0.4.6
+LABEL io.airbyte.version=0.4.7
```

Comment:
Not sure if this is right.

Reply:
This seems right since this is a bug fix but it's not a [user-facing functionality in a backwards compatible manner](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors)

Reply:
Ah ok I wasn't sure if this actually fell under a major version change
>data that was previously being synced will no longer be synced

Reply:
If anything the checkpointing changes for JDBC connectors should have been a major version bump 😰